dist/bin/dotc

```diff
@@ -89,10 +89,6 @@ case "$1" in
    -nobootcp) unset bootcp && shift ;;
       -colors) colors=true && shift ;;
    -no-colors) unset colors && shift ;;
-      -jrebel) jrebel=true && shift ;;
-   -no-jrebel) unset jrebel && shift ;;
-
-      -toolcp) require_arg classpath "$1" "$2" && toolcp="$2" && shift 2 ;;
```

Comment:
We are currently not supporting `toolcp` and it's not properly passed in `bin/dotc`. So I remove it for now, and add it back when we have `toolcp` support.